src/components/spot/storelist/selectCategory.tsx

```diff
@@ -1,13 +1,21 @@
 import styled from 'styled-components';
 import { categories } from './category';
+import { ClickedLocation } from '@provider/ClickedLocation';
 
 interface Props {
   setCategory: React.Dispatch<React.SetStateAction<string>>;
+  setClickedLocation?: React.Dispatch<
+    React.SetStateAction<ClickedLocation | undefined>
+  >;
 }
+const SelectCategory = ({ setCategory, setClickedLocation }: Props) => {
+  const handleChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    setCategory(e.target.value);
+    if (setClickedLocation) setClickedLocation(undefined);
+  };
 
-const SelectCategory = ({ setCategory }: Props) => {
   return (
-    <Selector onChange={(e) => setCategory(e.target.value)}>
+    <Selector onChange={(e) => handleChange(e)}>
```

Comment:
소소하게 아래처럼 적어주셔도 좋겠네요 ㅎㅎ

```suggestion
    <Selector onChange={handleChange}>
```